pandas/io/excel/_base.py

```diff
@@ -1217,14 +1219,15 @@ class ExcelFile:
            - Otherwise if ``path_or_buffer`` is in xlsb format,
              `pyxlsb <https://pypi.org/project/pyxlsb/>`_ will be used.
 
-           .. versionadded:: 1.3.0
+       .. versionchanged:: 1.3.0
```

Comment:
Please show screenshots, this looks to me as you changed the format here

Reply:
Will do, in the meantime have a look here, the layout is really messed up https://pandas.pydata.org/docs/reference/api/pandas.read_excel.html?highlight=read_excel#pandas.read_excel